root/static/scripts/guess-case/MB/GuessCase/Handler/Base.js

```diff
@@ -709,42 +705,40 @@ MB.GuessCase.Handler.Base = function (gc) {
                     } else {
                         break numberloop;
                     }
+                // Look for a group of 3 digits
+                } else if (gc.i.matchWordAtIndex(subIndex, gc.re.DIGITS_TRIPLE)) {
```

Comment:
Changes in https://github.com/metabrainz/musicbrainz-server/pull/1270/commits/c36a6737c93bc116c374698a2e9783b9aa27a078 undeniably make the code clearer but this change as it moves comment into the previous code block with unusual indentation. I’m not sure ESLint has a rule to prevent that, and `no-lonely-if` has no option.

In this particular case, the comment doesn’t seem useful anyway, maybe just remove it?